huntsman/camera/pyro.py

```diff
@@ -684,3 +688,65 @@ def focuser_move_to(self, position):
 
     def focuser_move_by(self, increment):
         return self._camera.focuser.move_by(increment)
+
+# Filterwheel methods - these are used by the remote filterwheel client,
+# huntsman.filterwheel.pyro.FIlterWheel
+
+    @property
+    def has_filterwheel(self):
+        return self._camera.filterwheel is not None
+
+    @property
+    def filterwheel_name(self):
+        return self._camera.filterwheel.name
+
+    @property
+    def filterwheel_model(self):
+        return self._camera.filterwheel.model
+
+    @property
+    def filterwheel_uid(self):
+        return self._camera.filterwheel.uid
+
+    @property
+    def filterwheel_is_connected(self):
+        return self._camera.filterwheel.is_connected
+
+    @property
+    def filterwheel_is_moving(self):
+        return self._camera.filterwheel.is_moving
+
+    @property
+    def filterwheel_is_ready(self):
+        return self._camera.filterwheel.is_ready
+
+    @property
+    def filterwheel_filternames(self):
+        return self._camera.filterwheel.filter_names
+
+    @property
+    def filterwheel_position(self):
+        return self._camera.filterwheel.position
+
+    @property
+    def filterwheel_current_filter(self):
+        return self._camera.filterwheel.current_filter
+
+    @property
+    def filterwheel_is_unidirectional(self):
+        return self._camera.filterwheel.is_unidirectional
+
+    def filterwheel_move_to(self, position):
+        self._camera.filterwheel._move_to(position)
+
+    def filterwheel_event_set(self):
+        self._camera.filterwheel._move_event.set()
+
+    def filterwheel_event_clear(self):
+        self._camera.filterwheel._move_event.clear()
+
+    def filterwheel_event_is_set(self):
+        return self._camera.filterwheel._move_event.is_set()
+
+    def filterwheel_event_wait(self, timeout=None):
+        return self._camera.filterwheel._move_event.wait(timeout)
```

Comment:
```suggestion
        return self._camera.filterwheel._move_event.wait(timeout)

```

End with a blank line.